common/templates/emails/default_post_enrollment_message.txt

```diff
@@ -1,19 +1,17 @@
-Hi, %%USER_FULLNAME%%!
+Hi %%USER_FULLNAME%%!
 
-Thank you for enrolling in %%COURSE_DISPLAY_NAME%%.  You are receiving this
-email from the Stanford Online Team as a confirmation that you have
-successfully enrolled in %%COURSE_DISPLAY_NAME%%.  The course will
-now appear on your Stanford Online (https://lagunita.stanford.edu/dashboard)
-dashboard.  This course is already under way, so make sure to sign in
-(https://lagunita.stanford.edu/login) and click "View Course" from your dashboard
-in order to get started.  Doing this will take you to the Course Info page for
-more information.
+Thank you for enrolling in %%COURSE_DISPLAY_NAME%%. You are receiving
+this email from the Stanford Lagunita Team as a confirmation that you have
```

Comment:
@tanyaxp wanna try moving `this` to the previous line and see what it does to the diff?